be/src/agent/task_worker_pool.cpp

```diff
@@ -717,7 +719,18 @@ void TaskWorkerPool::_publish_version_worker_thread_callback() {
                          << ", error_code=" << res;
             finish_task_request.__set_error_tablet_ids(error_tablet_ids);
         } else {
-            LOG(INFO) << "publish_version success. signature:" << agent_task_req.signature;
+            for (int i = 0; i < succ_tablet_ids.size(); i++) {
+                TabletSharedPtr tablet =
+                        StorageEngine::instance()->tablet_manager()->get_tablet(succ_tablet_ids[i]);
+                if (tablet != nullptr) {
```

Comment:
should we always do small compaction after PublishVersionTask?

Reply:
> should we always do small compaction after PublishVersionTask?

 not necessary ，changed to  batch submit